packages/graphql/src/types.ts

```diff
@@ -166,6 +166,7 @@ export type CustomScalarField = BaseField;
 export interface CustomEnumField extends BaseField {
     // TODO Must be "Enum" - really needs refactoring into classes
     kind: string;
+    defaultValue?: any;
```

Comment:
**nitpick (blocking)** I may be wrong, but for an enum field it looks like this is always going to be of type string, so maybe we can use a stricter type here?

```suggestion
    defaultValue?: string;
```